<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Child process</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>1.5 STDOUT</string>
    </property>
    <widget class="QPushButton" name="pushButtonSendCout">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>371</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send Text to STDOUT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Text for STDOUT:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditCout">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>3 WM_COPYDATA</string>
    </property>
    <widget class="QLabel" name="labelMsgCopyData">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Message from Parent</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>411</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>4.1 File mapping</string>
    </property>
    <widget class="QPushButton" name="pushButtonFMCreate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFMClose">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLoadFileMapping">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditFileMapping">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>281</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelFileMappingInfo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>371</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>INFO:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSaveFileMapping">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>411</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>4.2 File mapping with file</string>
    </property>
    <widget class="QTextEdit" name="textEditFMFile">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFMFCreate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFMFClose">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFMFLoad">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFMFSave">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelFileMappingFInfo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>371</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>INFO:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>434</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
